### DISCUSSION TITLE
Markdownlint + Reviewdog + Automerge

### DISCUSSION THREAD

**Discussion by calexandre** [nosportugal/terraforming-the-cloud-part1#3] [Ideas]
Que acha deste workflow @pfilourenco ? Poderoso não achas? :D 

Tens mais ideias que gostarias de ver "automatizadas"? (tb tou a testar este forum lol)

**Comment by pfilourenco** [comment]
Eu tenho algumas ideias, mas acho que é mais para o tema das pipelines. xD
Aqui para o github ainda não me surgiu mais nada. (tenho de explorar mais)

**Reply by calexandre** [reply]
isto ta muito forte...estou surpreendido com a panoplia de features que aqui andam...

**Reply by pfilourenco** [reply]
sim, sem duvida. também não fazia ideia.
até vi uma github action para criar o README de modulos de terraform, depois fui ver que a google usa isso xD

**Reply by calexandre** [reply]
eish isso é mesmo mt forte! Tens q experimentar e partilhar... fiquei curioso :)
